event-sourcing/event-source/src/main/java/uk/gov/justice/services/eventsourcing/source/core/exception/AggregateCreationException.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.justice.services.eventsourcing.source.core.exception;
+
+public class AggregateCreationException extends RuntimeException {
+
+    public static final String ACCESS_ERROR = "Unable to create aggregate due to access error";
```

Comment:
After discussing with Shaun again decided to remov the messages and moved to  DefaultSnapshotService and constructor just has only one constructor for message and exception